koan/utils.py

```diff
@@ -325,12 +328,12 @@ def get_vm_state(conn, vmid):
 
 def check_dist():
     """
-    Determines what distro we're running under.
+    Determines what distro we're running under (with the distro module).
     """
-    if os.path.exists("/etc/debian_version"):
-        import lsb_release
-        return lsb_release.get_distro_information()['ID'].lower()
-    elif os.path.exists("/etc/SuSE-release"):
+    distroname = distro.id()
+    if distroname is "debian":
+        return distroname
+    elif distroname is "suse" or distroname is "sles" or "opensuse" in distroname:
```

Comment:
Please use `distro.like()` and check if "suse" is in there. That would be better!

Reply:
Unfortunately this isn't working for Debian. They don't have anything in `distro.like()`. So you'll have to keep the name for them and only use like for SUSE.